dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -820,3 +820,12 @@ CWWKD1081.entity.general.err.explanation=An error prevents the \
  specified entity from being used.
 CWWKD1081.entity.general.err.useraction=Locate the entity and make \
  corrections if needed.
+
+CWWKD1082.entity.classes.missing=CWWKD1082E: The persistence unit for the \
```

Comment:
The persistence unit for the {0} data store does not define one or more entity classes ({1}) that one or more repository interfaces ({2}) require. Repository interfaces use the data store.

Reply:
By splitting out "Repository interfaces use the data store", it becomes unclear that it's intending to refer specifically to the repository interfaces that are listed.  I'll start with your first sentence and change it to something else that hopefully will be acceptable.